### PR TITLE
WIP: Websocket example

### PR DESCRIPTION
**Pull request by eliska-n**

### CODE REVIEW

File: examples/websocket_server.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+		super().__init__()
+
+		# Initialize WebService
+		self.add_module(asab.web.Module)
+		self.WebService = self.get_service("asab.WebService")
+		self.WebContainer = asab.web.WebContainer(self.WebService, "web")
+
+		# Create remote connection endpoint
+		web_app = self.WebContainer.WebApp
+		web_app.router.add_get("/rc", self.rc)
```
Comment (ateska) [inline examples/websocket_server.py:24]: `/rc` is likely too off-context here.

**Review by ateska — CHANGES_REQUESTED** [review]

File: examples/websocket_server.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+		# To store future websocket connection
+		self.WS = None
```
[the remark this answers is not in the document]
Comment (eliska-n) [reply examples/websocket_server.py:27]: Yes, creating a WS container in asab has been in my backlog for a while... :(

**Comment by eliska-n** [comment]
@ateska 
I have a task to send continuously state of a long-running subprocess (docker compose up). But I didn't know how to send it to UI. And I thought that WebSocket would probably be the choice even for UI. But as we have already discussed, WebSockets with various purposes should be somehow separated, and it would be nice to have a service in asab that would help me to handle this. And as this pull request has already been open but somehow useless, I decided to give it a bigger role. 
I have doubts about the code, but I would like to open the discussion.

**Comment by ateska** [comment]
Ha!
This is brave ;-)

File: asab/web/ws/container.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+class Message:
+	id: str
+	msg: bytes
+
+
+class WebSocketContainer():
+	def __init__(self, ws, id, svc):
+		self.ID = id
+		self.WS = ws
+		self.Messages = list()
```
Comment (ateska) [inline asab/web/ws/container.py:20]: Meh, that's not a good idea :-/

**Review by ateska — CHANGES_REQUESTED** [review]
We will need to elaborate on this a bit ...

**Comment by eliska-n** [comment]
I fixed the `get()` method and added `send_json_all()`. And I like it.

**Review by ateska — APPROVED** [review]
Ok. So let's merge it ;-)

**Pull request merged**